.github/workflows/issue-resolver.yml

```
@@ -0,0 +1,130 @@
+            ISSUE_DATA=$(echo $issue | base64 --decode)
+            ISSUE_NUMBER=$(echo $ISSUE_DATA | jq -r '.number')
+            ISSUE_TITLE=$(echo $ISSUE_DATA | jq -r '.title')
+            ISSUE_BODY=$(echo $ISSUE_DATA | jq -r '.body')
+            ISSUE_AUTHOR=$(echo $ISSUE_DATA | jq -r '.author.login')
+            
+            echo "Processing issue #$ISSUE_NUMBER: $ISSUE_TITLE"
+            
+            # Check if it's a bot issue
+            if [[ "$ISSUE_AUTHOR" == "pixeebot" ]] || [[ "$ISSUE_AUTHOR" == "github-actions" ]] || [[ "$ISSUE_TITLE" == *"Sentry"* ]] || [[ "$ISSUE_TITLE" == *"Sonar"* ]] || [[ "$ISSUE_TITLE" == *"Security"* ]]; then
+              echo "Issue #$ISSUE_NUMBER is a bot issue - auto-resolving"
+              
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)